### ISSUE TITLE
Cantaro PersName

### ISSUE THREAD

**Issue by msb81** [setriplette/golden_age#16]
@setriplette do you want me to tag <persName> in this case or leave it blank?

<persName>Doña Marina</persName>

Hola me llamo...

**Comment by msb81** [comment]
Also, did anyone edit my xml file? my oXygen is red and im not sure why because last time i left it, it was green. lol

**Comment by setriplette** [comment]
When you tag people, don't tag the character names from the play.

When I opened your Cántaro xml file after I pulled from Github, it was red, but I think I turned it green to do the reading view. I did make a couple of changes since then to your front matter, but that shouldn't have turned it red. I can take a look at it later!

Tag `<persName type="hist">` for historical people and <persName type="myth"> for Greco-Roman or other legendary figures, like Apollo, the sibyls, King Arthur, etc.

If you're in doubt over what category a person belongs to, just open an issue on github.

For places, tag `<placeName>`. Examples of these would be Madrid, Spain, Constantinople, etc. Don't tag places you couldn't find geocoordinates for, like the farm, the city, the street, inside the house, etc. No generic places, only specific.

**Comment by msb81** [comment]
Yeah I just opened it and it was already red. Not sure what's wrong.

Get Outlook for iOShttps://aka.ms/o0ukef

**Comment by setriplette** [comment]
I'm going to pull right now and take a look.

**Comment by setriplette** [comment]
What error are you getting? Mine is green. I would maybe pull again?

**Comment by setriplette** [comment]
If it's "content not allowed in prologue" that's not going to keep you from working with it--we've had that issue a lot with macs and it might be invisible characters or it might be a problem with the scheme line, but you can go ahead and keep coding if that's all.

**Comment by setriplette** [comment]
Oh, and make sure you are looking at the right file. The one Becca pushed is a mess because of PC special character issues (for once, advantage Macs). You want Moza_tei.xml

**Comment by ebeshero** [comment]
@setriplette Hmmm...that special characters thing actually shouldn't be an issue in oXygen...if the characters are munged, they are simply not Unicode forms, and need to be. Want me to take a look and see what is going on?

**Comment by setriplette** [comment]
The special characters issue came from when Becca copied and pasted the original texts from wikisource. Nothing untoward going on with that now, but Becca's file hung around in our github for a while and I got rid of it last week I think. I'm not sure what Maddie's seeing because I'm all green.

**Comment by msb81** [comment]
Hey @setriplette @ebeshero  I have a merge conflict when I try to pull anything from golden age, any idea how to resolve it?

**Comment by setriplette** [comment]
Do you have changes you want to keep? If not, reset to my last commit. I don't quite remember the command--Becca or Dr. B?

**Comment by setriplette** [comment]
I actually managed to pull your changes, so they don't conflict with mine.

**Comment by msb81** [comment]
Maybe i fixed it lol

**Comment by setriplette** [comment]
Ugh, something is up with your file! It is now all red.

**Comment by msb81** [comment]
Last time i checked it, it was green. What is going on?! Ugh.

**Comment by setriplette** [comment]
Did you do an identity transform? There's a lot of HEAD 
elements with file names and numbers

**Comment by msb81** [comment]
I dont even know what that is lol

**Comment by msb81** [comment]
If i did then i dont know how

**Comment by setriplette** [comment]
We are probably going to need to revert to a previous commit--I don't know how to fix this any other way

**Comment by msb81** [comment]
It's telling me the speech elements arent allowed

**Comment by msb81** [comment]
Okay, we'll fix it when we come back I guess!

**Comment by setriplette** [comment]
I rolled back to a previous commit and it removed the weird code.
Save any of your javascript files you want to keep in a different place, and then go in your git and do

git reset --hard `4f2bee31696dbab7e6efd78edc72116b34005508`

This long number is my commit number from earlier.

That will get you back to my previous commit and to a version of moza without the weird code. Try just doing the hand-coding this weekend. If you do Java Script 2, duplicate the moza file and work on it outside of our github folder. My guess is that working on that homework led you do do some xslt that added that stuff to the file. Just a guess!

**Comment by ebeshero** [comment]
@setriplette Nice work repairing that on your own! I wasn't looking at my e-mail and missed this. 
@msb81 Are you still in a state of merge conflict? I can see if I can investigate what happened to the code: My guess is that the weird code you saw is something to do with your attempt to resolve the conflict...
